templates/site/index.html

```diff
@@ -7,7 +7,7 @@
                     capability.",
                     "Marble is the bridge between intricate climate data and the people who seek to
                     understand it, fostering a community of inquiry and discovery in the realm of climate science."] %}
-
+{% set banner_video = "video/pexels-mikhail-nilov-8318649 (1440p).mp4" %}
```

Comment:
Since this is the only page that sets this variable, all other pages have a plain grey block at the top.
I think we want there to be no banner/video on pages other than home, right?

Reply:
I'll leave the decision to Shruti.

Reply:
@ShrutiK100

Reply:
Shruti said all other pages will have a static banner image.  An overlay (the gray block) may still be needed, depending on the colours in the static image.